code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -248,6 +248,36 @@
 	glass_quality = 1
 	shealth = 10
 
+	// Special snowflake code for building glass floors.
+	/obj/item/stack/sheet/glass/rglass/afterattack(atom/Target, mob/user, adjacent, params)
```

Comment:
I like this new style of indentation.
almost artistic, or maybe autistic I'm not 100% sure.